### PR TITLE
Update env vars so review apps work

### PR DESCRIPTION
**Pull request by alanmoo**
Closes https://github.com/mozilla/network-pulse/issues/730

### CODE REVIEW

File: pulseapi/settings.py [hunk 1]
```diff
@@ -25,6 +25,8 @@
     DEBUG=(bool, False),
     USE_S3=(bool, False),
     SSL_PROTECTION=(bool, False),
+    CORS_REGEX_WHITELIST=(tuple, ()),
```
Comment (cadecairos) [inline pulseapi/settings.py]: I believe that if you specify this as a tuple, comma delimited strings are automatically converted to the right format, so you shouldn't need to call `split(',')` on the variable below (if you use `env()` not `os.getenv()`)

File: pulseapi/settings.py [hunk 2]
```diff
@@ -165,18 +167,19 @@
...
 
 # we want to restrict API calls to domains we know
-CORS_ORIGIN_ALLOW_ALL = False
+if '*' in env('CORS_WHITELIST'):
+    CORS_ORIGIN_ALLOW_ALL = True
+else:
+    # and we want origin whitelisting
+    CORS_ORIGIN_WHITELIST = os.getenv(
+        'CORS_ORIGIN_WHITELIST',
+        'localhost:3000,localhost:8000,localhost:8080,test.example.com:8000,test.example.com:3000'
```
Comment (cadecairos) [inline pulseapi/settings.py]: can't this default be specified in an env file? Then we can just use `env()` to grab the setting from the environment.

File: pulseapi/settings.py [hunk 3]
```diff
@@ -165,18 +167,19 @@
...
+    ).split(',')
+    CORS_ORIGIN_REGEX_WHITELIST = env('CORS_REGEX_WHITELIST')
```
Comment (cadecairos) [inline pulseapi/settings.py]: we should look into how this works, we could simple specify a regex matching the review app domains that are created.

File: pulseapi/settings.py (hunk 1 above)
Comment (alanmoo) [reply pulseapi/settings.py]: Ok, I was just copying from the network site settings 😁

**Comment by alanmoo** [comment]
I simplified this PR, just the regex should do the trick now

**Pull request merged**